lmfdb/templates/workshops.html

```diff
@@ -6,6 +6,35 @@
 
 <ul>
 
+<li>
+Modular curves workshop 2, <br>
```

Comment:
Might as well include a link to the website: https://math.mit.edu/~edgarc/MCW2.html